static/js/poll_widget.js

```diff
@@ -64,7 +72,7 @@ export class PollData {
 
             options.push({
                 option: obj.option,
-                names: people.safe_full_names(voters),
+                names: this.render_safe_names_for_users(voters),
```

Comment:
I think we should eliminate the `render_` prefix, since we're really not rendering.  The `people.safe_full_names` function converts `[alice.user_id, bob.user_id]` to "Alice, Bob".  The `safe_` prefix in that name refers to the fact that we will just ignore any invalid users (such as expired users).  Let's come up with a good name for this in the poll context, and then that can possibly inspire the same name change for `people.safe_full_names`.